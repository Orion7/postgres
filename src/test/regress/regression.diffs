*** /home/orion/postgres/src/test/regress/expected/create_index.out	2015-05-15 02:42:10.918397426 +0300
--- /home/orion/postgres/src/test/regress/results/create_index.out	2015-06-08 21:19:03.181003929 +0300
***************
*** 1697,1702 ****
--- 1697,1704 ----
  SET enable_indexscan = OFF;
  SET enable_bitmapscan = ON;
  CREATE INDEX intarrayidx ON array_index_op_test USING gin (i);
+ NOTICE:  REGISTER.slot = 0, gen = 1
+ WARNING:  dynamic shared memory leak: segment 2020744785 still referenced
  explain (costs off)
  SELECT * FROM array_index_op_test WHERE i @> '{32}' ORDER BY seqno;
                       QUERY PLAN                     
***************
*** 1943,1948 ****
--- 1945,1952 ----
  (1 row)
  
  CREATE INDEX textarrayidx ON array_index_op_test USING gin (t);
+ NOTICE:  REGISTER.slot = 0, gen = 2
+ WARNING:  dynamic shared memory leak: segment 467587877 still referenced
  explain (costs off)
  SELECT * FROM array_index_op_test WHERE t @> '{AAAAAAAA72908}' ORDER BY seqno;
                           QUERY PLAN                         
***************
*** 2147,2152 ****
--- 2151,2158 ----
  -- And try it with a multicolumn GIN index
  DROP INDEX intarrayidx, textarrayidx;
  CREATE INDEX botharrayidx ON array_index_op_test USING gin (i, t);
+ NOTICE:  REGISTER.slot = 0, gen = 3
+ WARNING:  dynamic shared memory leak: segment 1785348097 still referenced
  SELECT * FROM array_index_op_test WHERE i @> '{32}' ORDER BY seqno;
   seqno |                i                |                                                                 t                                                                  
  -------+---------------------------------+------------------------------------------------------------------------------------------------------------------------------------
***************
*** 2233,2238 ****
--- 2239,2246 ----
  CREATE TABLE array_gin_test (a int[]);
  INSERT INTO array_gin_test SELECT ARRAY[1, g%5, g] FROM generate_series(1, 10000) g;
  CREATE INDEX array_gin_test_idx ON array_gin_test USING gin (a);
+ NOTICE:  REGISTER.slot = 0, gen = 4
+ WARNING:  dynamic shared memory leak: segment 1646118036 still referenced
  SELECT COUNT(*) FROM array_gin_test WHERE a @> '{2}';
   count 
  -------
***************
*** 2245,2250 ****
--- 2253,2260 ----
  --
  CREATE INDEX gin_relopts_test ON array_index_op_test USING gin (i)
    WITH (FASTUPDATE=on, GIN_PENDING_LIST_LIMIT=128);
+ NOTICE:  REGISTER.slot = 0, gen = 5
+ WARNING:  dynamic shared memory leak: segment 18400605 still referenced
  \d+ gin_relopts_test
       Index "public.gin_relopts_test"
   Column |  Type   | Definition | Storage 

======================================================================

*** /home/orion/postgres/src/test/regress/expected/gin.out	2015-05-15 02:42:10.922397440 +0300
--- /home/orion/postgres/src/test/regress/results/gin.out	2015-06-08 21:19:10.725022667 +0300
***************
*** 6,11 ****
--- 6,13 ----
  -- Create and populate a test table with a GIN index.
  create table gin_test_tbl(i int4[]);
  create index gin_test_idx on gin_test_tbl using gin (i) with (fastupdate = on);
+ NOTICE:  REGISTER.slot = 0, gen = 6
+ WARNING:  dynamic shared memory leak: segment 1339674024 still referenced
  insert into gin_test_tbl select array[1, 2, g] from generate_series(1, 20000) g;
  insert into gin_test_tbl select array[1, 3, g] from generate_series(1, 1000) g;
  vacuum gin_test_tbl; -- flush the fastupdate buffers

======================================================================

*** /home/orion/postgres/src/test/regress/expected/tsearch.out	2015-05-15 02:42:10.934397482 +0300
--- /home/orion/postgres/src/test/regress/results/tsearch.out	2015-06-08 21:19:11.849025463 +0300
***************
*** 151,156 ****
--- 151,158 ----
  RESET enable_seqscan;
  DROP INDEX wowidx;
  CREATE INDEX wowidx ON test_tsvector USING gin (a);
+ NOTICE:  REGISTER.slot = 0, gen = 7
+ WARNING:  dynamic shared memory leak: segment 614070057 still referenced
  SET enable_seqscan=OFF;
  SELECT count(*) FROM test_tsvector WHERE a @@ 'wr|qh';
   count 
***************
*** 1095,1100 ****
--- 1097,1104 ----
  -- test finding items in GIN's pending list
  create temp table pendtest (ts tsvector);
  create index pendtest_idx on pendtest using gin(ts);
+ NOTICE:  REGISTER.slot = 0, gen = 10
+ WARNING:  dynamic shared memory leak: segment 1402283618 still referenced
  insert into pendtest values (to_tsvector('Lore ipsam'));
  insert into pendtest values (to_tsvector('Lore ipsum'));
  select * from pendtest where 'ipsu:*'::tsquery @@ ts;

======================================================================

*** /home/orion/postgres/src/test/regress/expected/jsonb.out	2015-05-15 02:42:10.922397440 +0300
--- /home/orion/postgres/src/test/regress/results/jsonb.out	2015-06-08 21:19:11.937025682 +0300
***************
*** 1888,1893 ****
--- 1888,1895 ----
  (1 row)
  
  CREATE INDEX jidx ON testjsonb USING gin (j);
+ NOTICE:  REGISTER.slot = 0, gen = 8
+ WARNING:  dynamic shared memory leak: segment 1529664583 still referenced
  SET enable_seqscan = off;
  SELECT count(*) FROM testjsonb WHERE j @> '{"wait":null}';
   count 
***************
*** 1964,1969 ****
--- 1966,1973 ----
  
  -- array exists - array elements should behave as keys (for GIN index scans too)
  CREATE INDEX jidx_array ON testjsonb USING gin((j->'array'));
+ NOTICE:  REGISTER.slot = 0, gen = 9
+ WARNING:  dynamic shared memory leak: segment 1034002165 still referenced
  SELECT count(*) from testjsonb  WHERE j->'array' ? 'bar';
   count 
  -------
***************
*** 2074,2079 ****
--- 2078,2085 ----
  --gin path opclass
  DROP INDEX jidx;
  CREATE INDEX jidx ON testjsonb USING gin (j jsonb_path_ops);
+ NOTICE:  REGISTER.slot = 0, gen = 11
+ WARNING:  dynamic shared memory leak: segment 327180090 still referenced
  SET enable_seqscan = off;
  SELECT count(*) FROM testjsonb WHERE j @> '{"wait":null}';
   count 

======================================================================

