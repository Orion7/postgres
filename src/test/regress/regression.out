test tablespace               ... ok
parallel group (20 tests):  oid name varchar pg_lsn float4 txid char text regproc int2 boolean money uuid bit float8 int4 int8 enum rangetypes numeric
     boolean                  ... ok
     char                     ... ok
     name                     ... ok
     varchar                  ... ok
     text                     ... ok
     int2                     ... ok
     int4                     ... ok
     int8                     ... ok
     oid                      ... ok
     float4                   ... ok
     float8                   ... ok
     bit                      ... ok
     numeric                  ... ok
     txid                     ... ok
     uuid                     ... ok
     enum                     ... ok
     money                    ... ok
     rangetypes               ... ok
     pg_lsn                   ... ok
     regproc                  ... ok
test strings                  ... ok
test numerology               ... ok
parallel group (20 tests):  comments path lseg reltime timetz tinterval box abstime time polygon date circle macaddr point tstypes line inet interval timestamp timestamptz
     point                    ... ok
     lseg                     ... ok
     line                     ... ok
     box                      ... ok
     path                     ... ok
     polygon                  ... ok
     circle                   ... ok
     date                     ... ok
     time                     ... ok
     timetz                   ... ok
     timestamp                ... ok
     timestamptz              ... ok
     interval                 ... ok
     abstime                  ... ok
     reltime                  ... ok
     tinterval                ... ok
     inet                     ... ok
     macaddr                  ... ok
     tstypes                  ... ok
     comments                 ... ok
parallel group (6 tests):  geometry regex type_sanity horology oidjoins opr_sanity
     geometry                 ... ok
     horology                 ... ok
     regex                    ... ok
     oidjoins                 ... ok
     type_sanity              ... ok
     opr_sanity               ... ok
test insert                   ... ok
test create_function_1        ... ok
test create_type              ... ok
test create_table             ... ok
test create_function_2        ... ok
parallel group (2 tests):  copyselect copy
     copy                     ... ok
     copyselect               ... ok
parallel group (2 tests):  create_operator create_misc
     create_misc              ... ok
     create_operator          ... ok
parallel group (2 tests):  create_view create_index
     create_index             ... FAILED
     create_view              ... ok
parallel group (11 tests):  create_cast create_aggregate drop_if_exists create_function_3 typed_table vacuum create_table_like constraints inherit triggers updatable_views
     create_aggregate         ... ok
     create_function_3        ... ok
     create_cast              ... ok
     constraints              ... ok
     triggers                 ... ok
     inherit                  ... ok
     create_table_like        ... ok
     typed_table              ... ok
     vacuum                   ... ok
     drop_if_exists           ... ok
     updatable_views          ... ok
test sanity_check             ... ok
test errors                   ... ok
test select                   ... ok
parallel group (20 tests):  select_having case select_distinct_on select_distinct random namespace hash_index delete select_implicit update select_into btree_index union portals subselect arrays aggregates join prepared_xacts transactions
     select_into              ... ok
     select_distinct          ... ok
     select_distinct_on       ... ok
     select_implicit          ... ok
     select_having            ... ok
     subselect                ... ok
     union                    ... ok
     case                     ... ok
     join                     ... ok
     aggregates               ... ok
     transactions             ... ok
     random                   ... ok
     portals                  ... ok
     arrays                   ... ok
     btree_index              ... ok
     hash_index               ... ok
     update                   ... ok
     namespace                ... ok
     prepared_xacts           ... ok
     delete                   ... ok
parallel group (11 tests):  lock security_label replica_identity collate gist brin gin rowsecurity privileges spgist matview
     brin                     ... ok
     gin                      ... FAILED
     gist                     ... ok
     spgist                   ... ok
     privileges               ... ok
     security_label           ... ok
     collate                  ... ok
     matview                  ... ok
     lock                     ... ok
     replica_identity         ... ok
     rowsecurity              ... ok
parallel group (4 tests):  async psql alter_generic misc
     alter_generic            ... ok
     misc                     ... ok
     psql                     ... ok
     async                    ... ok
test rules                    ... ok
test event_trigger            ... ok
parallel group (19 tests):  combocid advisory_lock portals_p2 xmlmap tsdicts functional_deps guc equivclass json dependency tsearch jsonb window cluster select_views foreign_data bitmapops indirect_toast foreign_key
     select_views             ... ok
     portals_p2               ... ok
     foreign_key              ... ok
     cluster                  ... ok
     dependency               ... ok
     guc                      ... ok
     bitmapops                ... ok
     combocid                 ... ok
     tsearch                  ... FAILED
     tsdicts                  ... ok
     foreign_data             ... ok
     window                   ... ok
     xmlmap                   ... ok
     functional_deps          ... ok
     advisory_lock            ... ok
     json                     ... ok
     jsonb                    ... FAILED
     indirect_toast           ... ok
     equivclass               ... ok
parallel group (19 tests):  limit prepare conversion xml plancache returning temp copy2 without_oid sequence rowtypes truncate with domain polymorphism rangefuncs largeobject alter_table plpgsql
     plancache                ... ok
     limit                    ... ok
     plpgsql                  ... ok
     copy2                    ... ok
     temp                     ... ok
     domain                   ... ok
     rangefuncs               ... ok
     prepare                  ... ok
     without_oid              ... ok
     conversion               ... ok
     truncate                 ... ok
     alter_table              ... ok
     sequence                 ... ok
     polymorphism             ... ok
     rowtypes                 ... ok
     returning                ... ok
     largeobject              ... ok
     with                     ... ok
     xml                      ... ok
test stats                    ... ok
